riptide-parent/pom.xml

```diff
@@ -132,6 +135,23 @@
                 <version>${project.version}</version>
             </dependency>
 
+            <!-- Jakarta -->
```

Comment:
Riptide as a whole doesn't depend on web services, it's just a soap module and auto configure. Please keep dependencies on their respective places in appropriate pom files

Reply:
in this case this compile time dependencies should go directly to riptide-soap module, and appropriate api declaration with test scope in spring auto configure

Reply:
I added it to dependency management in the parent pom. Actually dependency consumption is happening at `riptide-soap` project.

Reply:
this is not appropriate place for plugin dependency of the single plugin